ckan/logic/action/create.py

```diff
@@ -227,7 +227,7 @@ def package_create(context, data_dict):
     return_id_only = context.get('return_id_only', False)
 
     if return_id_only:
-        return context['id']
+        pkg.id
```

Comment:
Missing a `return`?

Reply:
Scarily this worked, so I'm gueesing this isn't an existing test for this :(

Reply:
Good catch, I just fixed the test to correctly check the return value (`assert` vs. `assert_equals`)